Brewfile

```diff
@@ -1,4 +1,9 @@
 brew 'python@2'
+brew 'yarn'
```

Comment:
Should we actually do this? I know we install a specific version of yarn.

Reply:
The [docs](https://docs.sentry.io/internal/environment/) don't specify a version: `brew install nodejs yarn` (btw `yarn` implies `nodejs` so I didn't include that) but our Travis config pins it to 1.3.2.

Reply:
It's not too big of a deal, especially for local dev, but we should probably update our travis + getsentry to use 1.6.0 as well